integration/tests/collection/suite_test.go

```diff
@@ -66,11 +66,14 @@ func TestCollectorSuite(t *testing.T) {
 func (suite *CollectorSuite) SetupTest(name string, nNodes, nClusters uint) {
 	var (
 		conNode = testnet.NewNodeConfig(flow.RoleConsensus, testnet.WithLogLevel(zerolog.ErrorLevel), testnet.AsGhost())
-		exeNode = testnet.NewNodeConfig(flow.RoleExecution, testnet.WithLogLevel(zerolog.FatalLevel), testnet.AsGhost())
-		verNode = testnet.NewNodeConfig(flow.RoleVerification, testnet.WithLogLevel(zerolog.FatalLevel), testnet.AsGhost())
+		// DKG require at least 2 consensus nodes
+		conNode2 = testnet.NewNodeConfig(flow.RoleConsensus, testnet.WithLogLevel(zerolog.ErrorLevel), testnet.AsGhost())
```

Comment:
The collection integration tests was failing because the bootstrap step needs the random source to determine the cluster assignment, but to get the random source, it needs to get the DKG size, however, previously there was only 1 consensus node, which won't be able to return a correct DKG size. 

Adding another consensus node resolves the problem.

Reply:
It's possible to simulate the random beacon with 1 node only (DKG + threshold signature). I remember this was done in some earlier integration tests. 
Let me know if that's still needed and we can fix it, otherwise working with at least 2 nodes is fine.

Reply:
It was [failing here](https://github.com/onflow/flow-go/blob/0ab49815b6fae8a890f1c287f35604ab5bec2bf6/crypto/bls_thresholdsign.go#L150-L154), where the [`ThresholdSignMinSize` is `2`](https://github.com/onflow/flow-go/blob/0ab49815b6fae8a890f1c287f35604ab5bec2bf6/crypto/types.go#L95), and dkg.Size() was returning `1` with only 1 node.